daemon/Dockerfile

```diff
@@ -21,6 +21,9 @@ RUN mkdir -p /etc/container_environment
 ADD 2.sh /etc/runit/2
 RUN chmod +x /etc/runit/2
 
+ADD populate.sh /populate.sh
```

Comment:
added the script file. I had merged everything in one PR originally and have been reworking it all to split bug fixes and features.